Managers/ConsoleMan.cpp

```diff
@@ -253,6 +255,16 @@ void ConsoleMan::PrintString(string toPrint)
 {
     // Add the input line to the console
     m_pConsoleText->SetText(m_pConsoleText->GetText() + "\n" + toPrint);
+    if(m_LogToCli)
+    {
+        std::regex re_error("(ERROR|SYSTEM):");
```

Comment:
Why the snake casing on re_error, re_path and re_name? Why not just go with errorRegex, pathRegex, nameRegex or regexError, regexPath, regexName? Or is this some norm for regex I'm unaware of?
Also, I find this a bit dense to read, would be nice to have space between sections. Combined with the commenting style changes I suggested up above, you'd have something like this

```
// Colour the words error and system red
std::regex re_error("(ERROR|SYSTEM):");
toPrint = std::regex_replace(toPrint, re_error, "\033[1;31m$&\033[0;0m");

// Colour path strings green
std::regex re_path("\\w*\\.rte\\/(\\w| |\\.|\\/)*(\\/|\\.bmp|\\.wav|\\.lua|\\.ini)");
toPrint = std::regex_replace(toPrint, re_path, "\033[1;32m$&\033[0;0m");

// Colour name strings yellow
std::regex re_name("(\"[A-Z].*\"|\'[A-Z].*\')");
toPrint = std::regex_replace(toPrint, re_name, "\033[1;33m$&\033[0;0m");

std::cout << "\r" << toPrint << std::endl;
```

I know it's a bit longer, but I think it makes it a lot clearer.